test-data/single-instruction-assignment.yaml

```diff
@@ -51,3 +51,49 @@ code:
 post:
   - r1.type=number
   - r1.value=0
+---
+test-case: stack assign immediate
+
+pre: ["r10.type=stack", "r10.offset=512"]
+
+code:
+  <start>: |
+    *(u64 *)(r10 - 8) = 0
+
+post:
+  - r10.type=stack
+  - r10.offset=512
+  - s[504...511].value=0
```

Comment:
Should `s[504...511].type = number` also show up here?

Reply:
I have decided not to show that, since it should be shown in the "numbers" bitset version of the stack. So the bug is that it is not shown as part of the `post` (I think I was not sure where and how to put it).

Reply:
I have decided not to show that, since it should be shown in the "numbers" bitset version of the stack. So the bug is that it is not shown as part of the `post` (I think I was not sure where and how to put it).